docs/File-Formats.md

```diff
@@ -608,6 +612,13 @@ The MAF format recognized by the portal (excluding the annotation columns alread
 34. **t_ref_count (Optional)**: Reference allele count (tumor).
 35. **n_alt_count (Optional)**: Variant allele count (normal).
 36. **n_ref_count (Optional)**: Reference allele count (normal).
+37. **cbp_driver_passenger (Optional)**: "PUTATIVE_PASSENGER" or "PUTATIVE_DRIVER".
+38. **cbp_driver_passenger_annotation (Optional)**: field to give more information about the cbp_driver_passenger value (limited to 80 characters). This field can only be present if the cbp_driver_passenger is also present in the MAF file. This field is free text. Example values for this field are: "Pathogenic" or "VUS".
+39. **cbp_driver_class (Optional)**: free label/category that marks the mutation as a putative driver (limited to 20 characters). This field is free text. Example values for this field are: "Tier 1", Tier 2"...
+40. **cbp_driver_class_annotation (Optional)**: field to give more information about the cbp_driver_class value (limited to 80 characters). This field can only be present if the cbp_driver_class is also present in the MAF file. This field is free text. Example values for this field are: "Highly Actionable", "Potentially Actionable", "Not currently actionable"...
```

Comment:
I think it's important to explain the ordering of these classes. If you would have these 3 examples values, a user wants them in this specific order.